package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "aat-sdk",
+  "version": "1.0.0",
```

Comment:
Change this to the actual version which should be `0.0.1` for the initial package publish on NPM.